dist/vl-select.js

```diff
@@ -24,17 +24,32 @@ Promise.all([
 * @property {boolean} data-vl-select-search-result-limit - Attribuut om het aantal resultaten te limiteren.
 * @property {boolean} data-vl-select-search-no-result-limit - Attribuut om het aantal resultaten te limiteren.
 * @property {boolean} data-vl-select-deletable - Attribuut om te activeren of deactiveren dat het geselecteerde kan verwijderd worden.
+* @property {string} data-vl-search-placeholder - Attribuut bepaalt de placeholder van het zoek adres input element.
+* @property {string} data-vl-search-no-results-text - Attribuut bepaalt de tekst wanneer er geen zoekresultaten meer zijn.
+* @property {string} data-vl-no-more-options - Attribuut bepaalt de tekst wanneer er geen keuzes meer mogelijk zijn.
 *
 * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-select/releases/latest|Release notes}
 * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-select/issues|Issues}
 * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-select.html|Demo}
 */
 export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelectElement)) {
+  static get DEFAULT_SEARCH_PLACEHOLDER() {
+    return 'Zoek item';
+  }
+
+  static get DEFAULT_SEARCH_NO_RESULT() {
+    return 'Geen resultaten gevonden';
```

Comment:
we spreken in de default teksten nu over item, resultaten en opties, miss allemaal als item benoemen

Reply:
De teksten komen uit een file in de vl-ui-core (src/js/i18n/translations/nl.js). Zo zijn ze nu default en daarom is momenteel voor deze waarden gekozen.